AUTHORS

```diff
@@ -0,0 +1,5 @@
+Anísio Mandlate
+Edson Michaque
+Elton Laice
+Nélio Macombo
+Rosário P. Fernandes
```

Comment:
Maybe we should also put contact information here, like:
...
Edson Michaque <edson@michaque.com>
...

Reply:
Sounds good. Can you share your email contact? I'll ask the other authors as well.

Reply:
Mine is edson@michaque.com

Reply:
We can ask them to make the change themselves :-D

Reply:
@edsonmichaque hehe, makes sense to me!